src/multiselect_contextmenu.js

```diff
@@ -837,13 +930,279 @@ const updateToMultiCopyToBackpack =
       Blockly.ContextMenuRegistry.registry.register(copyToBackpack);
     };
 
+/**
+ * Modification for context menu 'commentDelete' to
+ * be available for multiple comments.
+ */
+const registerCommentDelete = function() {
+  const deleteOption = {
+    displayText: function(scope) {
+      let count = 0;
+      const workspace = scope.comment.workspace;
+      const dragSelection = dragSelectionWeakMap.get(workspace);
+      dragSelection.forEach(function(id) {
+        const comment = workspace.getCommentById(id);
+        if (comment) {
+          // Count the number of blocks that are nested in this block.
+          count += 1;
+        }
+      });
+      return (count <= 1) ?
+          Blockly.Msg['DELETE_COMMENT'] :
+          Blockly.Msg['DELETE_X_COMMENTS'].replace('%1', String(count));
```

Comment:
We don't have `DELETE_X_COMMENTS` in https://github.com/google/blockly/blob/8323642d0eb677648f004f294208d2734c6e8d65/msg/messages.js#L98 So we should have another alternative here (similar to your code below):

```js
        return Blockly.Msg['DUPLICATE_X_COMMENTS']?
            Blockly.Msg['DUPLICATE_X_COMMENTS'].replace(
                '%1', count) :
            Blockly.Msg['DUPLICATE_COMMENT'] + ' (' +
            count + ')';
```